tests/models/__init__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
Since there is no logic needed in this file, we can remove it since we are trying to respect PEP420.

Reply:
as discussed IRL: needed for VSCode test discovery to work